tests/test_grainstats_minicircle.py

```diff
@@ -44,4 +47,25 @@ def test_image_set(minicircle_grainstats: GrainStats, tmpdir, value, expected):
     assert Path.exists(Path(tmpdir) / "grains/minicircle" / "None_grain_image_0.png") == expected
     assert Path.exists(Path(tmpdir) / "grains/minicircle" / "None_grainmask_0.png") == expected
     assert Path.exists(Path(tmpdir) / "grains/minicircle" / "None_grain_image_0.png") == expected
-    assert Path.exists(Path(tmpdir) / "grains/minicircle" / "None_grain_image_0.png") == expected
\ No newline at end of file
+    assert Path.exists(Path(tmpdir) / "grains/minicircle" / "None_grain_image_0.png") == expected
```

Comment:
Sorry for not picking this up earlier but are these last two not the same?

Reply:
Yep absolutely! There should only be three assert tests here for each of the produced grain images. Resolved in commit 87891c7